testsuite/tests/asmcomp/is_static_flambda.ml

```diff
@@ -91,3 +91,26 @@ let () =
     try (failwith [@inlined always]) "some other string" with exn -> exn
   in
   assert(is_in_static_data exn)
+
+(* Verify that approximation intersection correctly loads exported
+   approximations.
+
+   Is_static_flambda_dep.cpl is a couple with 1 as first element. The
+   intersection of approximations should return a block of size 2 and
+   tag 0 containing 1 in the first field.
```

Comment:
I'm not sure this description is quite right, although maybe it's just the wording being confusing.  The intersection of approximations yields `[tag 0: [tag 0: Int 1, Unknown], Unknown]`, no?